docs/reference/accelerometer.md

```diff
@@ -99,6 +107,8 @@ public class Accelerometer extends Device {
   public void disable();
   public int getSamplingPeriod();
   public double[] getValues();
+  public int getLookupTableSize();
```

Comment:
I am not sure these functions are really useful in Java, Python and MATLAB, as we can retrieve the array size easily:
**Java**
```java
double[] arr = getValues();
int size = arr.length / 3;
```
**Python**
```python
  arr = self.getLookupTable()
  size = len(arr) / 3
```
**MATLAB**
```matlab
  arr = wb_accelerometer_get_lookup_table();
  size = length(arr) / 3;
```
So, I would suggest to remove them.

Reply:
This is something I wasn't sure about, whether to keep it consistent or use features of the other languages. The same applies to C++, we could use `vector<double>`, but we decided to be consistent. However, for Java, Python and MATLAB it makes more sense to leave out `getLookupTableSize()`, I will delete them.

Reply:
I agree, it is less consistent than other languages, but I believe we should leverage on the capabilities of these languages to simplify our API. Users will have no problem figure out how to use these array length functions.
C++ is a special case as we do not use the vector class in our API so far. So introducing it only for these functions would be a big disruption for a little thing.